docs/pool-upgrade-how-to.md

```diff
@@ -98,68 +98,90 @@ application version:
    steps *(these must be done for all the nodes: validators, observers, seed
    nodes, sentry nodes)*:
 
-    1. Switches current user to the user on behalf of whom `cosmovisor` service
-       is running:
+   1. Make sure that the `auto-download` parameter is present in `cosmovisor.service` (in the folder - `/etc/systemd/system/`) and if the `auto-download` parameter does not exist, then add this `auto-download` parameter to the section `Service` for enabled **`auto-download`**.
+      
+      For example:
+
+      ```bash
+      Environment="DAEMON_ALLOW_DOWNLOAD_BINARIES=true"
+      ```
+      > **_Note:_**  In this case, `true` means that the **`auto-download`** is enabled. If you want to disable **`auto-download`**, you can set the value `false` (`DAEMON_ALLOW_DOWNLOAD_BINARIES=false`) or remove the above line of code from the `cosmovisor.service`.
+
+   2. Restart `cosmovisor.service`
+      ```bash
+      sudo systemctl restart cosmovisor
+      ```
+
+   3. Switches current user to the user on behalf of whom `cosmovisor` service
+      is running:
+
+      ```bash
+      su - <USERNAME>
+      ```
```

Comment:
Step 2 is required only if auto-download parameter has changed. So better to integrate it into step 1
It is better to keep "3. Switches current user to the user on behalf of whom `cosmovisor` service" as the 1st step

Reply:
Done! ✅